Source/ConversationV1/Models/ConversationModel.swift

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension Conversation {
+    
+    /// A Conversation model
+    public struct ConversationModel: JSONDecodable {
+        
+        /// The Conversation workspace application identifier
+        public var workspaceID: WorkspaceID?
```

Comment:
The change from ObjectMapper to Freddy for JSON deserialization allows us to be a lot more expressive with our models. In particular:

(1) Var/Try: ObjectMapper required all properties to be `var`. The `var` keyword specified that the properties might change in the future. With our SDK, though, we don't ever expect these properties to change after being set in the initializer. As a result, we can use `let` instead to express their immutability to the client. Using Freddy in this SDK, I have yet to see a reason for a model's properties to be declared as `var` instead of `let`--they should universally be declared using `let`.

2. Optionals: ObjectMapper required all properties to be optional. This is no longer the case with Freddy. Now we can *properly* express the properties that are required and the properties that are optional. Unfortunately, much of the Swagger documentation specified *all* properties as optional. This is almost never actually the case, though. Unless I have good reason to believe that a property is actually optional, I tend to declare it as non-optional and write good tests to see if it ever fails. (Freddy will produce a failure if the key for a supposedly non-optional property cannot be found.) Keep in mind that the `init` and the property declarations should match -- non-optional properties should use `try` and optional properties should use `try?`.

Long-story short: these properties look like they are actually non-optional. So you can remove the `?` from their type. (If they were actually optional, then you should update the `init` to use `try?` instead of `try`.)